src/components/header_menu.jsx

```diff
@@ -62,18 +63,20 @@ const HeaderMenu = ({
           <WithBreakpoints hideOn={Breakpoints.Mobile}>
             <NotePath
               drivePath={drivePath}
-              path={file.attributes.path || drivePath}
+              path={file.attributes.path || drivePath.split('#')[1]}
```

Comment:
why this change?

Reply:
It allows me to have the drive path without querying the actual drive directory. I found it to be pretty effective since it eliminates one http request

Reply:
OK why not, but I still don't understand why we need to use the drivePath for that. I don't understand why we need this `|| drivePath`? Does it mean that `file.attributes.path` can be empty? 🤔 

I don't think that relying on an internal stuff like `#` is a good way to handle it if needed. Right now you use `generateWebLink` but pretty soon we'll use an other method and the `#` will not be there anymore. This is something to avoid.